install_config/install/docker_registry.adoc

```diff
@@ -947,15 +947,15 @@ auth:
 ====
 
 [[docker-registry-configuration-reference-middleware]]
-==== middleware
+==== Middleware
 
-The **repository** middleware extension allows to configure OpenShift middleware
-responsible for interaction with OpenShift and image proxying.
+The *repository* middleware extension allows to configure {product-title}
+middleware responsible for interaction with {product-title} and image proxying.
 
 [NOTE]
 ====
-The **middleware** section may not be overridden using environment variables.
-There are few exceptions though.
+The *middleware* section may not be overridden using environment variables.
+There are a few exceptions, however.
```

Comment:
> Can you please provide some examples of specific exceptions here? Thanks!

@ahardin-rh sure. Currently it's just:

```
middleware:
  repository:
    - name: openshift
      options:
        acceptschema2: false
        enforcequota: false
        projectcachettl: 1m
```

Using following environment variables:

```
// EnforceQuotaEnvVar is a boolean environment variable that allows to turn quota enforcement on or off.
// By default, quota enforcement is off. It overrides openshift middleware configuration option.
// Recognized values are "true" and "false".
REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA

// ProjectCacheTTLEnvVar is an environment variable specifying an eviction timeout for project quota
// objects. It takes a valid time duration string (e.g. "2m"). If empty, you get the default timeout. If
// zero (e.g. "0m"), caching is disabled.
REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_PROJECTCACHETTL

// AcceptSchema2EnvVar is a boolean environment variable that allows to accept manifest schema v2
// on manifest put requests.
REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2
```